apache_postgresql/website_template.yml

```diff
@@ -0,0 +1,569 @@
+apiVersion: influxdata.com/v2alpha1
+kind: Label
+metadata:
+    name: modest_wu_5b4001
+spec:
+    color: '#009f5f'
+    name: website
+---
+apiVersion: influxdata.com/v2alpha1
+kind: Bucket
+metadata:
+    name: bettering_faraday_5b4005
+spec:
+    associations:
+      - kind: Label
+        name: modest_wu_5b4001
+    name: website
+    retentionRules:
+      - everySeconds: 2.592e+06
```

Comment:
what does this translate too?

Reply:
30 days

Reply:
not crazy about our `everySeconds` rule... thinking out loud here. Would be cool to allow something like the following if you ask me. WDYT?

```
- every: 30
  period: day
```

Reply:
I personally think that the actual content of the template file should be largely ignored by users and developers alike, the interface with them should be at the tools or UI level, so I'm not especially worried about how we serialize retention periods in the file.

Reply:
that's fair. I speaking more to the API of influxdb here than the actual pkg itself. What you see here is a reflection of the current API. I only just happened to dawn on me this API was like that when I saw your 30 day in seconds number 😬.

Reply:
if you're going to change it, could you use the same syntax as Flux? So something like:
```
- every: 30d
- every: 24h
```

Reply:
yah, that'd be even better 👍